.github/workflows/release.yaml

```diff
@@ -27,4 +27,40 @@ jobs:
           args: release --rm-dist
           workdir: ./cmd/manager/ # <-- path to the directory containing the main() function
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  container-images:
+    name: Build and push container images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo and checkout submodules
+        uses: actions/checkout@v2.3.4
+        with:
+          submodules: recursive
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.16"
+      - name: Prepare
+        id: prep
+        run: |
+          VERSION=sha-${GITHUB_SHA::8}
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF/refs\/tags\//}
+          fi
+          echo ::set-output name=BUILD_DATE::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+          echo ::set-output name=VERSION::${VERSION}
+          echo ::set-output name=BUILDARCH::$(go env GOARCH)
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.3.0
+      - name: Log in to quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Build and push (multi-arch)
+        run: |
+          make docker-buildx VERSION=${{ steps.prep.outputs.VERSION }} .
```

Comment:
I think you have forgotten to use: docker-build-multi & docker-push-multi

Reply:
I think we can use buildx to solve this for us, https://blog.oddbit.com/post/2020-09-25-building-multi-architecture-im/
What do you think?

Reply:
`make docker-buildx` does use `docker buildx` already 😄. The non-buildx `make` targets like `docker-build-multi` are there because of this caveat I mentioned in the PR:

> docker buildx build --load cannot be used with a multi-arch image, so there are separate Makefile targets to build each arch without pushing them for testing, and then to combine them in a manifest.

Basically, the local docker daemon can’t load a multi-arch image manifest, only individual images. So for testing PR’s, I included the other `make` targets to build without buildx.

Reply:
Thinking about this a bit more, another workaround would be to [run a Docker registry as a `service`](https://docs.github.com/en/actions/guides/about-service-containers) in the PR workflow. Then we could [use `docker/build-push-action`](https://github.com/docker/build-push-action/blob/master/docs/advanced/multi-platform.md) without having to push every PR to a real registry, and still be able to scan the image in the next step.

Reply:
Okay, didn't understand that part.
Is it worth testing each container image per architecture using trivy, the packages **should** be the same between the different architectures. Would this make the CI simpler? If not lets keep it as is.

I still think it's worth building each container so we don't introduce some bad changes and find them after we have merged.

Added a minor comment about hadoolint further down, when that is fixed I guess the CI will continue and we will see if this works now, then there is probably no need to change it.